### ISSUE TITLE
Source for the solar irradiance web services and data

### ISSUE THREAD

**Issue by dbutrovski** [NICTA/aremi-natmap#19]
The solar irradiance web services currently consumed into AREMI and the NEII Viewer are originating from Geoscience Australia. These include GHI mean, GHI minimum and DNI mean. The BoM is the custodian for solar irradiance data and it will be desirable for related web services, including future updates to the data, to come from the custodian, source agency. The web services currently used are created for ASEIS application back in 2013. GA will want to decommission ASEIS, along with the solar irradiance services (data behind which is not maintained in ASEIS context).

**Comment by meh9** [comment]
Thanks for the note, @dbutrovski.

The new Bureau services were just published last week, I have briefly tested them and we have started the work on integrating these. We are hoping that we will have them in the next release, in which case the ASEIS layers will be removed.

**Comment by meh9** [comment]
@stevage, relevant issue for the NEII viewer.

**Comment by stevage** [comment]
Yep, the SI layers have already been removed from it (by BOM).

**Comment by meh9** [comment]
Done - I've replaced the ASEIS layers with the Bureau of Meteorology Solar Climatology layers, see commit f2d4024e931395c022fbb0184b74e2f2a12d1b1b

I did not add the two Bureau the hourly layers. For the two hourly layers there is too little variation, and there is only 24 hours of data on the 15th of each month. We need to do something about the way we render these, see:

https://github.com/TerriaJS/terriajs/issues/896

We should also speak to the Bureau about the hourly layers, as the colour variation between hours of data is unfortunately so minor as to be nearly completely unnoticeable.
